Modules/Filtering/DistanceMap/test/itkHausdorffDistanceImageFilterTest.cxx

```diff
@@ -92,7 +93,7 @@ itkHausdorffDistanceImageFilterTest(int, char *[])
     filter->Update();
     filter->Print(std::cout);
```

Comment:
Is this print left on purpose?

Reply:
Good catch. Will address that later tonight.

Reply:
Done in 3b36da7.